### PR TITLE
Zh/cross shard

### PR DESCRIPTION
**Pull request by sman2013**
for cross shard transaction

### CODE REVIEW

File: src/chain_spec.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 use {
     primitives::{
-        sr25519, Pair,
+        ed25519, sr25519, Pair,
```
Comment (guoxbin) [inline src/chain_spec.rs]: do not change chain_spec.rs

File: src/service/foreign_demo.rs [hunk 2]
```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2019 Yee Foundation.
```
Comment (guoxbin) [inline src/service/foreign_demo.rs]: do not include foreign_demo.rs

File: src/service.rs [hunk 3]
```diff
@@ -192,14 +192,21 @@ construct_service_factory! {
...
                     config,
                     foreign_network_wrapper,
                     service.client(),
-                    executor,
+                    executor.clone(),
                 )?;
-                // TODO: register to transaction pool
+
+                // relay-transfer
+				yee_relay::start_relay_transfer::<Self, _, _>(
```
Comment (guoxbin) [inline src/service.rs]: check indent

**Review by guoxbin — CHANGES_REQUESTED** [review]

File: src/chain_spec.rs (hunk 1 above)
Comment (sman2013) [reply src/chain_spec.rs]: it's ok

**Pull request merged**